clientd3d/music.c

```diff
@@ -647,7 +801,16 @@ void MusicAbort(void)
 void MusicStart(void)
 {
    if (state == STATE_GAME)
-      PlayMusicRsc(bg_music);
+   {
+      if (latest_music == bg_music)
```

Comment:
I don't understand this change... a comment would be good here.

Reply:
I added a comment.  The idea is the user has turned on music in the config.  We need to determine which song type to play in that instance.